seatunnel-e2e/seatunnel-flink-connector-v2-e2e/pom.xml

```diff
@@ -36,6 +36,7 @@
         <module>connector-fake-flink-e2e</module>
         <module>connector-mongodb-flink-e2e</module>
         <module>connector-iceberg-flink-e2e</module>
+        <module>connector-iceberg-hadoop3-flink-e2e</module>
```

Comment:
Add module: connector-iceberg-hadoop3-spark-e2e

Or

move module to https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e